### DISCUSSION TITLE
Calendar Different colors

### DISCUSSION THREAD

**Discussion by MrCasquette** [directus/directus#17069] [Archive]
I'm Sorry if the question was already answered but I'd tried to find an answer and dont't found anything. Is there a way to change the color of the different item of the calendar. For example I want to make a planning for my company. And I need to distinguish between different employees. For example one appears blue, another red and another yellow. Making it more visually that just a name or a simple string on the same colors. Thank you for the reply.

**Comment by rijkvanzanten** [comment]
There's currently no way to adjust the color per event, though I love that idea. Mind opening a new feature request discussion outlining how you think that should ideally work? It could either be as a `color` field on the item itself, or maybe there's some sort of conditional formatting option on the calendar layout where you can say "Make event color X when field A = Y"

**Reply by MrCasquette** [reply]
Sorry I had no notifications about your answer. Thank you @rijkvanzanten. You're always awesome. I will open a new feature request

**Comment by ramstein74** [comment]
I´m interested also in having colors in calendar. It should be a must !!
